### PR TITLE
adds IncentivesIcon

### LINKED ISSUE #10646: [Icon]: New icon <incentive>
### How will these icons be used?

image

This icon will be utilized to indicate to a user they may be eligible for an incentive of some kind within their Shopify account.  These may be limited time offers, or an "always-on" incentive that shows when a user meets specific criteria to be eligible for a special offer. Incentives are often price-related, but may also include non-monetary incentives.  All incentives communicated to a user represent a boost to the user's current account in some way, whether it be a credit towards their subscription cost, special access to a trial or feature, or any other form of elevating, accelerating, or boosting their account.

We are proposing a new icon for this use case as we have been borrowing/repurposing an existing icon (star) to serve this purpose in our early exploration of incentives. We do not wish to dilute the meaning of the star as it is also utilized in favoriting or other actions across their Shopify experience, and instead want experiences related to incentives to be specific and easily identifiable as something the user can gain additional value from.

[Incentives Iconography visual exploration figma](https://www.figma.com/file/AKPuD0P1cvo2IQiZANnGlT/Monetization-visual-system-framework?type=design&node-id=554%3A327&mode=design&t=bqB90VzdEew7YRGh-1)

[Incentives as a system WIP figma](https://www.figma.com/file/09VnNrKJFKi3HyDvIUmMI8/Incentives-as-a-system?type=design&node-id=471%3A29041&mode=design&t=mNqN6H2LMGttjBJ6-1)

### Icon name

incentive

### What type of icon set?

Major (20x20)

### Icon description

An accented upward arrow.

image

This is intended to represent a boost, upgrade, accelerant, or growth opportunity
Wayfinding: used to direct merchants to opportunities for earning, claiming, or simply informing them of additional value/functionality available in their Shopify account.
Concept: used to represent a boost, elevation, or upgrade of sorts.

Examples of use:
image

image

### Icon keywords

incentives, incentive, growth, 

### Icon author

Jess Bottali, José Torre, Thomas Jonkajtys, Bryce Coster

### Icon SVG code

```shell
<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" fill="none"><path fill="#1F2124" fill-rule="evenodd" d="M9.966 4.59a.288.288 0 0 0-.417 0L6.134 8.072a.323.323 0 0 0-.068.362c.05.126.15.195.277.195h1.458l1.491 7.118a.308.308 0 0 0 .296.253h.339a.308.308 0 0 0 .295-.253l1.491-7.118h1.46c.128 0 .227-.07.278-.195a.323.323 0 0 0-.069-.362L9.966 4.589ZM8.478 3.54a1.788 1.788 0 0 1 2.56-.001l3.415 3.482c1.108 1.13.353 3.108-1.28 3.108h-.241l-1.242 5.926A1.808 1.808 0 0 1 9.927 17.5h-.34a1.808 1.808 0 0 1-1.763-1.445l-1.241-5.926h-.24c-1.632 0-2.387-1.978-1.28-3.107l3.415-3.483ZM4.53 11.632a.75.75 0 0 1 .937.497l1.316 4.293a.75.75 0 1 1-1.434.44l-1.316-4.293a.75.75 0 0 1 .497-.937ZM14.929 11.62a.75.75 0 0 1 .55.907l-1.053 4.293a.75.75 0 0 1-1.457-.357l1.053-4.293a.75.75 0 0 1 .907-.55Z" clip-rule="evenodd"/></svg>
```

### Are there any other important details?

_No response_

### PR DESCRIPTION
**Pull request by jonarnaldo**
### WHY are these changes introduced?
Adds a new incentives icon to the Polaris Icons library as part of [US GPV Acquisition Incentive](https://vault.shopify.io/gsd/projects/37720)

closes https://github.com/Shopify/incentives/issues/1640

### WHAT is this pull request doing?
Adding `IncentivesMajor` so it can be used for any incentives notifications/banners etc.
image

### How to 🎩
n/a

### CODE REVIEW

**Comment by alex-page** [comment]
@jonarnaldo what is the urgency on this icon request?

**Comment by alex-page** [comment]
@jonarnaldo as discussed in Slack. We are about to ship a major release for icons (this week or next week). Adding this icon is going to be delayed until that icon ships. There are work arounds you can do in mobile and shopify/web to use this icon ASAP. When this ships we can come back and clean up the work arounds.

**Comment by BryceCoster** [comment]
Wanted to circle back and see if this is good to go yet? We'd love to get this up there as it unblocks [this ticket for our team](https://github.com/Shopify/incentives/issues/1640) and closes [this earlier icon request](https://github.com/Shopify/polaris/issues/10646)

**Comment by alex-page** [comment]
The major release of icons is released and we need to update this PR to follow the new naming conventions. We also removed version exclusive_use and set from icon .yml files.

Take a look here for more information: https://github.com/Shopify/polaris/pull/11138

**Review by alex-page — APPROVED** [review]

**Comment by heyjoethomas** [comment]
Please add the live, unflattened svg to [this file](https://www.figma.com/file/oeKyR0kuHIMktdVjdLHJvy/Polaris-Icons-(unflattened)?type=design&node-id=1-3&mode=design). Let me know when it's in there and I can approve.

**Review by sam-b-rose — APPROVED** [review]

File: .changeset/poor-crabs-think.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris-icons': minor
+---
+
+Adds IncentiveIcon
```
Comment (sam-b-rose) [inline .changeset/poor-crabs-think.md]:
```suggestion
Added `IncentiveIcon`
```

**Comment by cody-robinson-shopify** [comment]
@heyjoethomas - Icon I believe has been added to the figma!

image

**Comment by sam-b-rose** [comment]
@jonarnaldo this should be good to merge if you'd like! All ✅'s have passed

**Pull request merged**